### PR TITLE
Support file:// protocol (close #908)

### LINKED ISSUE #908: Support file:// protocol
Electron application usually uses file protocol for loading pages.
Also some application use absolute paths for resources, so we can't use simple static server for the proxying pages.
E.g. vscode uses absolute paths for scripts:
https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/electron-browser/bootstrap/index.js#L136
@inikulin @churkin What are you thinking about it?

### PR DESCRIPTION
**Pull request by LavrovArtem**
\cc @churkin @kirovboris

### CODE REVIEW

**Comment by kirovboris** [comment]
It's necessary provide the specifying of absolute url in tests:
```js
    //Win platforms
    fixture.page('C:/index.html');

    //*nix platforms
    fixture.page('/usr/home/index.html');
```

**Comment by LavrovArtem** [comment]
FPR

File: src/processing/resources/index.js [hunk 1]
```diff
@@ -5,12 +5,19 @@ import scriptProcessor from './script';
 import stylesheetProcessor from './stylesheet';
 import * as urlUtil from '../../utils/url';
 import { encodeContent, decodeContent } from '../encoding';
+import { platform } from 'os';
+
+const IS_WIN32 = platform() === 'win32';
```
Comment (kirovboris) [inline src/processing/resources/index.js]: I think just `IS_WIN`

File: src/processing/resources/index.js [hunk 2]
```diff
@@ -5,12 +5,19 @@ import scriptProcessor from './script';
...
+const DISK_RE  = /^[A-Za-z]:/;
 
 function getResourceUrlReplacer (ctx) {
     return function (resourceUrl, resourceType, charsetAttrValue, baseUrl) {
         if (!urlUtil.isSupportedProtocol(resourceUrl) && !urlUtil.isSpecialPage(resourceUrl))
             return resourceUrl;
 
+        if (IS_WIN32 && ctx.dest.protocol === 'file:' && DISK_RE.test(resourceUrl))
```
Comment (kirovboris) [inline src/processing/resources/index.js]: What is about *nix platforms? Can we at least add an unit test with hadling paths like `/usr/home/index.html`?

Comment (LavrovArtem) [inline src/processing/resources/index.js]: `/usr/home/index.html` is already relative path which can be resolved via the `url.resolve` function

Comment (kirovboris) [inline src/processing/resources/index.js]: I would add test anyway.

Comment (LavrovArtem) [inline src/processing/resources/index.js]: Sure)

**Pull request merged**